src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-setup.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: External services setup
+metaDescription: "Here are steps to enable and configure external services"
+---
+The external services feature is available for New Relic APM agents and OpenTelemetry. After you follow the steps below to set up external services, you may need to adjust the [sampling rate](#adjust-sampling) if you don't see enough data.
```

Comment:
Does this need a link to "OpenTelemetry"?

Reply:
@barbnewrelic, yes! I just added it! Thanks!